README.md

```diff
@@ -416,8 +416,12 @@ Special values:
 
 * `:required => true` Will display a red '*' to show it's required
 * `:scope => :the_scope` Will scope parameters in the hash, scoping can be nested. See example
+* `:method => :method_name` Will use specified method as a parameter value
 
-The value of scoped parameters can be set with both scoped (`let(:order_item_item_id)`) and unscoped (`let(:item_id)`) methods. It always searches for the scoped method first and falls back to the unscoped method.
+Retrieving of parameter value goes through several steps:
+1. if `method` option is defined and test case responds to this method then this method is used;
+2. if test case responds to scoped method then this method is used;
+3. overwise unscoped method is used.
```

Comment:
@oestrich @sineed This fallback won't work for some scenarios.

Example: 

```
get '/foo' do
  parameter :status, "Blah", method: :status_param

  example 'meh' do
    expect(status).to eq 200
  end
end
```
Even before entering the example block, rspec api documentation calls `status` to prepare the request, because `status_param` is not defined. When it calls `status`, it fails with `No response yet. Request a page first`, because the `status` method is only supposed to be called after a request is done.